sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/Diagnostics/AzureMonitorExporterEventSource.New.cs

```diff
@@ -49,5 +49,26 @@ public void TransmissionFailed(bool fromStorage, int statusCode, ConnectionVars
 
         [Event(8, Message = "{0} {1} {2}")]
         public void TransmissionFailed(string message, string retryDetails, string metaData) => WriteEvent(8, message, retryDetails, metaData);
+
+        [Event(9, Message = "{0} has been disposed.", Level = EventLevel.Verbose)]
```

Comment:
Should this be informational?

Reply:
I could go either way. The former was Verbose, and it was the only Verbose in our SDK.